pipeline/mrms_download.py

```diff
@@ -4,16 +4,23 @@
 from urllib.request import urlopen
 
 
-def download(inventory, dest_dir, max_download=4):
-    """Given an inventory file for a particular day, and a destination
-    directory, download all files in the inventory that are not
+def download(inventory, dest_dir = os.path.join(os.getcwd(),'Data'), max_download=4):
```

Comment:
Our data is already in a lower-case `data` directory. Should we change that name (and all current workflows) or should we change this?

Reply:
Change this, The object of this was to just add the data  directory as the default